vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1475,7 +1475,7 @@ - (void)onExcludedSitesChanged:(const std::string &)publisher_id exclude:(ledger
   }
 }
 
-- (void)onRemoveRecurring:(const std::string &)publisher_key callback:(ledger::RecurringRemoveCallback)callback
+- (void)onRemoveRecurring:(const std::string &)publisher_key callback:(ledger::RemoveRecurringTipCallback)callback
```

Comment:
Missing an update to L486:

```objc++
- (void)removeRecurringTipForPublisherWithId:(NSString *)publisherId
{
  ledger->RemoveRecurringTip(std::string(publisherId.UTF8String));
}
```

suggested update:
```objc++
- (void)removeRecurringTipForPublisherWithId:(NSString *)publisherId completion:(void (^)(BATResult))completion
{
  ledger->RemoveRecurringTip(std::string(publisherId.UTF8String), ^(ledger::Result result){
    completion(static_cast<BATResult>(result));
  });
}
```

and then in BATBraveLedger.h, find declaration for this method:

```objc++
- (void)removeRecurringTipForPublisherWithId:(NSString *)publisherId
```

to:

```objc++
- (void)removeRecurringTipForPublisherWithId:(NSString *)publisherId completion:(void (^)(BATResult result))completion
```

Reply:
done